### ISSUE TITLE
Allow resource move via drag-and-drop

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-template-editor#506]

**Comment by michelleif** [comment]
Yes it would be very nice, as well as being able to select multiple things and move them all at once. I have a bunch of stuff in a folder. I made a subfolder and want to move a sub-bunch of stuff into that subfolder. I have to move each one individually.
